grails-app/controllers/au/org/ala/profile/api/ApiController.groovy

```diff
@@ -510,6 +513,177 @@ class ApiController extends BaseController {
         }
     }
 
+    @GrantAccess
+    @Path("/api/opus/{opusId}/profile/{profileId}/image/{imageId}")
+    @Operation(
+            summary = "Get images associated with a profile",
+            operationId = "/api/opus/{opusId}/profile/{profileId}/image/{imageId}",
+            method = "GET",
+            responses = [
+                    @ApiResponse(
+                            responseCode = "200",
+                            content = @Content(
+                                    mediaType = "application/json",
+                                    schema = @Schema(
+                                            implementation = ImageListResponse.class
+                                    )
+                            ),
+                            headers = [
+                                    @Header(name = 'X-Total-Count', description = "Total number of images", schema = @Schema(type = "integer")),
```

Comment:
X-Total-Count will not be provided by this action.

Reply:
Changed